docs/platform/howto/use-azure-privatelink.rst

```diff
@@ -161,22 +161,69 @@ To enable Private Link access in `Aiven Console <https://console.aiven.io/>`_:
 
     Each service component can be controlled separately. For example, you can enable Private Link access for your Aiven for Apache Kafka® service, while allowing Kafka® Connect to only be connected via VNet peering.
 
-After toggling the values your Private Link resource will be rebuilt with load balancer rules added for the service component's ports. Connection information like the URI or hostname and port to access the service through the private endpoint is available on the service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_. 
+After toggling the values, your Private Link resource will be rebuilt with load balancer rules added for the service component's ports.
 
 .. note::
   
   For Aiven for Apache Kafka® services, the security group for the VPC endpoint must allow ingress in the port range ``10000-31000``. This is to accommodate the pool of Kafka broker ports used in the Private Link implementation.
 
+Acquire connection information
+------------------------------
+
+.. _one-privatelink-connection:
+
+One Azure Private Link connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have Azure Private Link enabled for one service component, you can preview its connection information (URI, hostname, or port required to access the service through the private endpoint) on the service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_.
```

Comment:
> If you have Azure Private Link enabled for one service component
I would rather say something like: "If you have one private endpoint connected to Aiven service". It doesn't depend on how many components but rather how many endpoints.